lib/presentation/ui/occurrences_time_widget.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+
+class OccurrencesTimeWidget extends StatelessWidget {
+
+  final DateTime _startTime;
+  final DateTime _endTime;
+  final DateTime _lastUpdated;
+
+  const OccurrencesTimeWidget(this._startTime, this._endTime, this._lastUpdated);
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      type: MaterialType.transparency,
+      child: new Container (
+                decoration: new BoxDecoration(
+                  color: Colors.grey,
+                  boxShadow: [new BoxShadow(
+                  color: Colors.black,
+                  blurRadius: 20.0,
+                  )],
+                ),
+                child:
+                    new Column(
+                      children: <Widget>[
+                        new Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: <Widget>[
+                            new Column(
+                              mainAxisSize: MainAxisSize.min,
+                              children: <Widget>[
+                                new Text(
+                                  "Início",
+                                  overflow: TextOverflow.clip,
+                                  style: DefaultTextStyle.of(context).style.apply(
+                                      fontSizeFactor: 0.5,
+                                      decoration: TextDecoration.none,
+                                      color: Colors.grey
+                                  ),
+                                ),
+                                new Text(
+                                  this._startTime != null ? this._startTime.hour.toString() + ":" + this._startTime.minute.toString() : "--",
+                                  overflow: TextOverflow.clip,
+                                  style: DefaultTextStyle.of(context).style.apply(
+                                      fontSizeFactor: 0.5,
+                                      decoration: TextDecoration.none,
+                                      color: Colors.lightGreen
+                                  ),
+                                ),
+                                new Text(
+                                  this._startTime != null ? this._startTime.day.toString() + "-" + this._startTime.month.toString() + "-" + this._startTime.year.toString() : "--",
+                                  overflow: TextOverflow.clip,
```

Comment:
Why are you clipping the datetime here?

Wouldn't it be best for the text to be flexible and, if needed, go to the the next line so that the user can see the correct time?

You might consider wrapping the Text widget inside another widget such as a Container to wrap the text

Reply:
@Vanethos , please look at my updated PR